languages/python/django-oso/django_oso/partial.py

```diff
@@ -91,17 +91,25 @@ def and_expr(expr: Expression, model: Model, **kwargs):
     return q
 
 
-def compare_expr(expr: Expression, _model: Model, path=(), **kwargs):
+def compare_expr(expr: Expression, model: Model, path=(), **kwargs):
     q = Q()
     (left, right) = expr.args
     left_path = dot_path(left)
     if left_path:
         return COMPARISONS[expr.operator](q, "__".join(path + left_path), right)
     else:
-        if isinstance(right, Model):
+        if isinstance(right, model):
             right = right.pk
         else:
+            # QUESTION: Should this just return an empty set? Similar to how invalid isa
+            # returns empty set.
             raise UnsupportedError(f"Unsupported comparison: {expr}")
```

Comment:
we should probably create a "constant" like `always_false = Q(pk__in=[])` at the top of the file and then use it in both places
```suggestion
			return Q(pk__in=[])
```

Reply:
I saw you guys added this constant in another PR, we can replace it then.